src/config.rs

```diff
@@ -5,12 +5,11 @@ const DEFAULT_DATABASE_URL: &str = "mongodb://localhost:27107";
 
 /// Get set database url
 pub fn get_database_url() -> String {
-    let database_url = std::env::var("DATABASE_URL").unwrap_or_else(|e| {
+    std::env::var("DATABASE_URL").unwrap_or_else(|e| {
         tracing::warn!(DEFAULT_DATABASE_URL, error = %e,
                 "Unable to retrieve API_PORT; falling back to default port");
```

Comment:
Should read "Unable to retrieve DATABASE_URL; falling back to default url" or similar. This wasn't added in this PR but I didn't notice it until now